diff --git a/weight.module b/weight.module
index e98642b..cf333ba 100644
--- a/weight.module
+++ b/weight.module
@@ -5,22 +5,6 @@
  * Provides a weight field that allows entities to be ordered.
  */
 
-function weight_menu() {
- $items['weight-test'] = array(
-   'title' => 'weight test',
-   'page callback' => 'weight_test',
-   'access callback' => TRUE,
-  );
-
-  return $items;
-}
-
-function weight_test() {
-  dpm(node_type_get_names());
-
-  return '';
-}
-
 /**
  * Implements hook_field_info().
  */
